Reference_Manual.md

```diff
@@ -1221,114 +1224,116 @@ You can also fine-tune this feature using `client_config::set_protperty` API as
+Once, we set the ssl context, the client will start with ssl enabled and try to connect to the server using the provided context.
 
+#### 6.1.2.1. Enabling Validation Of Server Certificate
+If you want to enable the validation of server certificate at the client side and disable any connection without a valid certificate, you can use the [context::set_verify_mode](https://www.boost.org/doc/libs/1_75_0/doc/html/boost_asio/reference/ssl__context/set_verify_mode.html) API. Here is an example:
+
+```c++
+    hazelcast::client::client_config config;
+
+    boost::asio::ssl::context ctx(boost::asio::ssl::context::method::tlsv12_client);
+    ctx.set_verify_mode(boost::asio::ssl::verify_peer);
+    ctx.set_default_verify_paths();
+
+    config.get_network_config().get_ssl_config().set_context(std::move(ctx));    
+    auto hz = hazelcast::new_client(std::move(config)).get();
 ```
-    /**
-     * Default protocol is tlsv12 and ssl is disabled by default
-     */
-    ssl_config ();
-    
-    /**
-     * Returns if this configuration is enabled.
-     *
-     * @return true if enabled, false otherwise
-     */
-    bool is_enabled() const;
-    
-    /**
-     * Enables and disables this configuration.
-     *
-     * @param enabled true to enable, false to disable
-     */
-    ssl_config  &set_enabled(bool enabled);
-    
-    /**
-     * Sets the ssl protocol to be used for this SSL socket.
-     *
-     * @param protocol One of the supported protocols
-     */
-    ssl_config  &set_protocol(ssl_protocol protocol);
-    
-    /**
-     * @return The configured SSL protocol
-     */
-    ssl_protocol get_protocol() const;
-    
-    /**
-     * @return The list of all configured certificate verify files for the client.
-     */
-    const std::vector<std::string> &get_verify_files() const;
-    
-    /**
-     * This API calls the OpenSSL SSL_CTX_load_verify_locations method underneath while starting the client
-     * with this configuration. The validity of the files are checked only when the client starts. Hence,
-     * this call will not do any error checking. Error checking is performed only when the certificates are
-     * actually loaded during client start.
-     *
-     * @param filename the name of a file containing certification authority certificates in PEM format.
-     */
-    ssl_config  &add_verify_file(const std::string &filename);
+
+The client will connect if the server has a certificate which is properly signed by wel known Certificate Authorities(CA). On Windows, you may need to set the well known CA Authorities file path since OpenSSL may not find the well known certifite authorities file. This can be solved by providing the [SSL_CERT_FILE](https://www.openssl.org/docs/man1.1.0/man3/SSL_CTX_set_default_verify_paths.html) environment variable set to point a file path that has all the needed well known certificate authorities. For example, you can get such a file maintained by Mozilla at https://curl.se/docs/caextract.html.
```

Comment:
```suggestion
The client will connect if the server has a certificate which is properly signed by well known Certificate Authorities(CA). On Windows, you may need to set the well known CA Authorities file path since OpenSSL may not find the well known certifite authorities file. This can be solved by providing the [SSL_CERT_FILE](https://www.openssl.org/docs/man1.1.0/man3/SSL_CTX_set_default_verify_paths.html) environment variable set to point a file path that has all the needed well known certificate authorities. For example, you can get such a file maintained by Mozilla at https://curl.se/docs/caextract.html.
```